R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+#' Accessibility panel
+#'
+#' @param dashboard_title Title of the host dashboard
+#' @param dashboard_url URL for the host dashboard
+#' @param repo_url URL for the dashboard repository
+#' @param non_accessible_components String vector containing a list of non accessible components
+#' @param specific_issues String vector containing descriptions of specific accessibility issues
+#' that have been identified as part of testing
+#' @param date_tested Date the application was last tested
+#' @param date_prepared Date the statement was prepared in it's current form
+#' @param date_reviewed Date the statement was last reviewed
+#' @param date_template_reviewed Date the underlying template was reviewed
+#' (default: 12th March 2024)
+#'
+#' @return shiny$tags$div element containing the html tags and content for the standard
+#' accessibility statement
+#' @export
+#'
+#' @examples
+#' a11y_panel(
+#'   "DfE Shiny template",
+#'   "https://department-for-education.shinyapps.io/dfe-shiny-template",
+#'   "25th April 2024",
+#'   "26th April 2024",
+#'   "2nd November 2024",
+#'   repo_url = "https://github.com/dfe-analytical-services/shiny-template",
+#' )
+a11y_panel <- function(
+    dashboard_title,
+    dashboard_url,
+    date_tested,
+    date_prepared,
+    date_reviewed,
+    date_template_reviewed = "12 March 2024",
+    repo_url = NA,
```

Comment:
NA gives an error, should it be `NULL` (mostly thinking out of consistency with other functions NULL seems to be more common than NA) and should we accept NULL silently in the function rather than throw an error?

![image](https://github.com/user-attachments/assets/59329d6b-3a4b-4c08-8a16-3b47e8f78f90)

Reply:
Switched to NULL (which is what I meant to do in the first place, don't know why I put NA!)